File: app/src/main/java/example/Handler.java
package example;

import com.amazonaws.services.lambda.runtime.Context;
import com.amazonaws.services.lambda.runtime.RequestHandler;
import com.amazonaws.services.lambda.runtime.LambdaLogger;

import java.util.Map;

public class Handler implements RequestHandler<Map<String, String>, String> {
    @Override
    public String handleRequest(Map<String, String> event, Context context) {
        LambdaLogger logger = context.getLogger();
        String response = "200 OK";
        // log execution details
        logger.log("ENVIRONMENT VARIABLES: " + System.getenv().toString());
        logger.log("CONTEXT: " + context);
        logger.log("EVENT: " + event.toString());
        logger.log("EVENT TYPE: " + event.getClass());
        return response;
    }
}

File: app/src/main/kotlin/example/Handler.kt
package example;

import com.amazonaws.services.lambda.runtime.Context
import com.amazonaws.services.lambda.runtime.RequestHandler

class Handler : RequestHandler<Map<String, String>, String> {

    override fun handleRequest(event: Map<String, String>, context: Context): String {
        val logger = context.logger
        val response = "200 OK";

        // log execution details
        logger.log("ENVIRONMENT VARIABLES: " + System.getenv().toString());
        logger.log("CONTEXT: $context");
        logger.log("EVENT: $event");
        logger.log("EVENT TYPE: " + event::class.java);
        return response;
    }
}
